### PR TITLE
fix(curriculum): Added quotations around 'Question #'

### LINKED ISSUE #46298: Quiz 33 - Quotes Around Value
### Describe the Issue

It would be clearer if we put quotes around the value we want the user to use.

### Affected Page

https://www.freecodecamp.org/learn/2022/responsive-web-design/learn-accessibility-by-building-a-quiz/step-33

### Your code

I have seen this mistaken answer a few times:
```css
p::before {
  content: Question #;
}
```

### Expected behavior

Suggested text:
To prevent unnecessary repetition, target the `before` pseudo-element of the `p` element, and give it a `content` property of `"Question #"`.

### Screenshots

_No response_

### System

All

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by casem725**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #46298

### CODE REVIEW

**Review by jeremylt — DISMISSED** [review]
LGTM. The double quotes match what's there in later steps

File: curriculum/challenges/english/14-responsive-web-design-22/learn-accessibility-by-building-a-quiz/6145ed1f22caab087630aaad.md [hunk 1]
```diff
@@ -17,10 +17,10 @@ You should use the `p::before` selector.
 assert.exists(new __helpers.CSSHelp(document).getStyle('p::before'));
 ```
 
-You should give the `p::before` pseudo-element a `content` property of `Question #`.
+You should give the `p::before` pseudo-element a `content` property of `"Question #"`.
 
 ```js
-assert.include(new __helpers.CSSHelp(document).getStyle('p::before')?.content, 'Question #');
+assert.include(new __helpers.CSSHelp(document).getStyle('p::before')?.content, '"Question #"');
```
Comment (jeremylt) [inline curriculum/challenges/english/14-responsive-web-design-22/learn-accessibility-by-building-a-quiz/6145ed1f22caab087630aaad.md:23]: woops, quick question to the more knowledgeable here - are the "" needed here on this line?

Comment (Sboonny) [reply curriculum/challenges/english/14-responsive-web-design-22/learn-accessibility-by-building-a-quiz/6145ed1f22caab087630aaad.md:23]: Nope, https://developer.mozilla.org/en-US/docs/Web/API/HTMLTemplateElement/content

**Review by ShaunSHamilton — DISMISSED** [review]
Requesting changes to block this until someone can confirm this does not break cases using single quotes etc. 

I am AFK for the rest of the day.

File: curriculum/challenges/english/14-responsive-web-design-22/learn-accessibility-by-building-a-quiz/6145ed1f22caab087630aaad.md (hunk 1 above)
Comment (Sboonny) [reply curriculum/challenges/english/14-responsive-web-design-22/learn-accessibility-by-building-a-quiz/6145ed1f22caab087630aaad.md:23]: You are right, I don't know why I included quotes in whitespace characters in my head 😅

> The white space is only replaced, when the getPropVal method is used, and only when true is passed as the second argument.

👍 

> Requesting changes to block this until someone can confirm this does not break cases using single quotes etc.

**Review by naomi-lgbt — DISMISSED** [review]
image

@ShaunSHamilton Single quotes seem to work.

**Comment by bbsmooth** [comment]
@nhcarrigan You just beat me too it (shakes fist). But then I also removed the double quotes from the assert and both single/double quotes worked as well. So it appears that we don't need the double quotes in the assert in the first place and should probably just get rid of them since they are already causing this much confusion.

**Review by naomi-lgbt — APPROVED** [review]
Passes locally with both quote styles.

**Review by Sboonny — APPROVED** [review]

**Comment by Sboonny** [comment]
> should probably just get rid of them since they are already causing this much confusion.

Although I am kinda in agreement, I am happy with the code below not pass. So all is good.

```css
p::before {
content: Question #;
}
```

**Comment by Sboonny** [comment]
Hi @casem725  . Congrats on your first pull request (PR)! 🎉

Thank you for your contribution to the page! 👍
We are happy to accept these changes and look forward to future contributions. 📝

**Review by moT01 — APPROVED** [review]

**Pull request merged**